src/constants/routes.js

```diff
@@ -1,4 +1,6 @@
 module.exports = {
+  resetPassword: '/reset',
```

Comment:
`reset-password` would be nice so it matches forgotPassword and is a little more clear

Reply:
👍